TEMPLATE-SPEC.md

```diff
@@ -9,69 +9,123 @@ Requires: spec-xyz | ip-xyz
 
 # Spec-<Number>: <Title>
 
-## Summary
+## Overview
 
-Background and short summary in layman terms.
+<!-- One page high-level overview; put details in the specification section and background in the previous section. Should be understandable by a new engineer or reader who isn't already working on the project. -->
 
-## Context
+Short summary in layman terms.
```

Comment:
let's be consistent with putting the section description in normal text or in the comment.

Reply:
OK, I prefer it as a comment.